tgym/envs/trading.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, data_generator, spread_coefficients, game_length=1000, tradin
         """
 
         assert data_generator.n_products == len(spread_coefficients)
+        assert history_length > 0
```

Comment:
👍  At some point, we should stop being lazy with `assert` and raise proper error messages.